lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -173,6 +176,17 @@ def add_preauth(xml, money, options)
         end
       end
 
+      def add_capture(xml, money, authorization, options)
+        currency = options[:currency] || currency(money)
+
+        xml.capture do
+          xml.transactionId          new_transaction_id
+          xml.referenceTransactionId authorization
+          xml.amount                 localized_amount(money, currency)
+          xml.currency               currency
```

Comment:
This can be made more concise by deleting line 180 and then copying line 158 above.

Reply:
I think the ```currency``` definition you mentioned deleting is needed since it is used for the amount as well.